lib/elixir/lib/io.ex

```diff
@@ -252,13 +286,14 @@ defmodule IO do
   @doc """
   Inspects `item` according to the given options using the IO `device`.
 
-  See `Inspect.Opts` for a full list of options.
+  See `inspect/2` for a full list of options.
   """
   @spec inspect(device, item, Keyword.t) :: item when item: var
   def inspect(device, item, opts) when is_list(opts) do
+    label  = if (label = opts[:label]), do: [to_chardata(label), ": "], else: ""
```

Comment:
Seems we're actually working with chardata here, and it would be better to rename `iodata` accordingly.